test/lein_monolith/plugin_test.clj

```diff
@@ -47,8 +47,6 @@
   (let [monolith (config/find-monolith! (read-example-project))
         subproject (-> (project/read "example/apps/app-a/project.clj")
                        (plugin/middleware monolith))]
-    (is (= '([amperity/greenlight "0.7.1"] ; This version of greenlight should come first so it'll take precedence over any other versions
-             [org.clojure/spec.alpha "0.3.218"]
-             [amperity/greenlight "0.6.0"]
-             [com.amperity/vault-clj "2.1.583"])
+    (is (= '([amperity/greenlight "0.7.1"]
+             [org.clojure/spec.alpha "0.3.218"])
```

Comment:
A passing note - Clojure considers vectors and lists to have the same equality class, meaning `(= [1 :two "three"] '(1 :two "three"))` => `true`. As a consequence, you'll almost always see code literals such as in unit tests like this use vectors instead of quoted lists. This has a number of advantages; one is that you can reference other values symbolically; another is you'll see somewhat cleaner error messages, instead of seeing the `(quote (,,,))` syntax expansion.